spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/StreamOperations.java

```diff
@@ -161,4 +161,19 @@ void updateStream(String streamName, String releaseName, PackageIdentifier packa
 	 * @param properties scale deployment properties.
 	 */
 	void scaleApplicationInstances(String streamName, String appName, Integer count, Map<String, String> properties);
+
+	/**
+	 * Retrieves all the applications' logs for the given stream name
+	 * @param streamName name of stream for which to get logs
+	 * @return logs of said stream
+	 */
+	String streamExecutionLog(String streamName);
```

Comment:
The Stream log changes need to be in a different PR.   The do not pertain to the current PR.

Reply:
OK